### PR TITLE
Jekoch/update e2e test

### PR DESCRIPTION
**Pull request by Jessica-Koch**

### CODE REVIEW

File: apps/teams-test-app/src/components/AppInitialization.tsx [hunk 1]
```diff
@@ -56,9 +56,12 @@ const NotifyExpectedFailure = (): React.ReactElement =>
           return;
         }
         const acceptableValues = Object.values(app.ExpectedFailureReason);
-        if (!input.reason && !acceptableValues.includes(input.reason)) {
+        if (input.reason && !acceptableValues.includes(input.reason)) {
           throw new Error(`input must be one of: ${JSON.stringify(acceptableValues)}`);
         }
+        if (!input.reason) {
+          input.reason = app.ExpectedFailureReason.Other;
+        }
```
Comment (anmaur) [inline apps/teams-test-app/src/components/AppInitialization.tsx:64]: Will it ever execute? Given that we have statements from 54-56

**Comment by erinha** [comment]
<p>Analyzed commit id : b08b4782a3c3b32d5660c33f3f410a824f3aea61</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -7 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>59.43 KB</td><td>59.42 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr><tr><td>Total Size</td><td>59.43 KB</td><td>59.42 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr></table></details><hr><p>Baseline commit: 247aa802c407b894cd515b6a838bf15db5b279a0</p>

File: apps/teams-test-app/src/components/AppInitialization.tsx (hunk 1 above)
Comment (Jessica-Koch) [reply apps/teams-test-app/src/components/AppInitialization.tsx]: omg you're right.....i think i just want vacation to come

Comment (Jessica-Koch) [reply apps/teams-test-app/src/components/AppInitialization.tsx]: I am so sorry for being sloppy, thank you for your patience

**Comment by erinha** [comment]
<p>Analyzed commit id : a28e1f7872d6aec9d1ec929b982a65a0010e58e2</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -7 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>59.43 KB</td><td>59.42 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr><tr><td>Total Size</td><td>59.43 KB</td><td>59.42 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr></table></details><hr><p>Baseline commit: 247aa802c407b894cd515b6a838bf15db5b279a0</p>

**Comment by erinha** [comment]
<p>Analyzed commit id : 4957b77b4447197368782a2ae4e8306284e0fb17</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -7 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>59.43 KB</td><td>59.42 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr><tr><td>Total Size</td><td>59.43 KB</td><td>59.42 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr></table></details><hr><p>Baseline commit: 247aa802c407b894cd515b6a838bf15db5b279a0</p>

File: apps/teams-test-app/src/components/AppInitialization.tsx [hunk 2]
```diff
@@ -52,11 +52,10 @@ const NotifyExpectedFailure = (): React.ReactElement =>
     onClick: {
       validateInput: input => {
         if (!input.reason) {
-          // this API actually allow for the input not to be provided
-          return;
+          input.reason = app.ExpectedFailureReason.Other;
         }
         const acceptableValues = Object.values(app.ExpectedFailureReason);
-        if (!input.reason && !acceptableValues.includes(input.reason)) {
+        if (input.reason && !acceptableValues.includes(input.reason)) {
```
Comment (anmaur) [inline apps/teams-test-app/src/components/AppInitialization.tsx:58]: Now do we even need this❔

**Review by anmaur — DISMISSED** [review]
With a minor suggestion

**Review by MarcinJuraszek — APPROVED** [review]

**Comment by erinha** [comment]
<p>Analyzed commit id : a9e7e5e536ffb59e487101d896ed30f589c40344</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -7 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>59.43 KB</td><td>59.42 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr><tr><td>Total Size</td><td>59.43 KB</td><td>59.42 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr></table></details><hr><p>Baseline commit: 247aa802c407b894cd515b6a838bf15db5b279a0</p>

**Comment by erinha** [comment]
<p>Analyzed commit id : 38372331e29b0be867401c521843f9328dc0ec0b</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -7 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>59.43 KB</td><td>59.42 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr><tr><td>Total Size</td><td>59.43 KB</td><td>59.42 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr></table></details><hr><p>Baseline commit: bd112e44bd54dff4b1b231e9709ce6490a1b5076</p>

**Comment by erinha** [comment]
<p>Analyzed commit id : c4737f87f9be22dbebda06254bea82b3b0b92d0c</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -7 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>59.43 KB</td><td>59.42 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr><tr><td>Total Size</td><td>59.43 KB</td><td>59.42 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr></table></details><hr><p>Baseline commit: bd112e44bd54dff4b1b231e9709ce6490a1b5076</p>

**Review by anmaur — APPROVED** [review]
LGTM, Thank you for considering the suggestion

**Comment by erinha** [comment]
<p>Analyzed commit id : 37777e279d99ad8da43288d476d62cef65e0a64b</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -7 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>59.38 KB</td><td>59.37 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr><tr><td>Total Size</td><td>59.38 KB</td><td>59.37 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr></table></details><hr><p>Baseline commit: 126f401aae255673ebb77e31804e0ee6bb2cb2f5</p>

**Pull request merged**